rllib/agents/mbmpo/model_vector_env.py

```diff
@@ -0,0 +1,92 @@
+import logging
```

Comment:
We should move this to `rllib/env/.`. It's not necessarily specific to MBMPO.

Reply:
Wondering, why we need this at all. Can't we just use the already existing vector wrapping functionality? Should the actually wrapped ModelEnv not expose the same API as any Env wrapped as a VectorEnv? In other words, the logic in this class here to call `self.model` to get the next obs (inside `vector_step`) should be abstracted inside the single ModelEnv, not here in the VectorizerWrapper. Let me know, if I'm not seeing something.

Reply:
Your first comment: I agree! This will be moved in the next commit, as soon as I add improved ensemble training and MAML dataset batching (like PPO minibatching), which hopefully result in identical performance to the original codebase. This will likely be committed tmrw.

Your second comment: It is more efficient to wrap it around a set of environments, rather than one enviornment. This is because it is not as efficient to call your policy `num_env` times versus 1 simple call to a batch of (obs, actions) from vec_env. This is a lot faster that way and is also done in the original codebase.

Reply:
Actually I will keep this in MBMPO rn, since it is hard to identify another MB algorithm that uses the specifics indicated in this file. (Dreamer is different)